Looks like the build is also failing due to a bad javadoc:
```
/__w/quilt-mappings/quilt-mappings/.gradle/temp/fakeSource/net/minecraft/item/ItemStack.java:709: error: reference not found
    * @see Item#onHit
```

--- a/mappings/net/minecraft/item/AnimalArmorItem.mapping
+++ b/mappings/net/minecraft/item/AnimalArmorItem.mapping
@@ -1,8 +1,12 @@
 CLASS net/minecraft/unmapped/C_bbljnksk net/minecraft/item/AnimalArmorItem
 	FIELD f_iimkkhka type Lnet/minecraft/unmapped/C_bbljnksk$C_rtwzapnt;
 	FIELD f_ludhudle texture Lnet/minecraft/unmapped/C_ncpywfca;
+	FIELD f_rvizqqlq overlayTexture Lnet/minecraft/unmapped/C_ncpywfca;
+	METHOD <init> (Lnet/minecraft/unmapped/C_cjzoxshv;Lnet/minecraft/unmapped/C_bbljnksk$C_rtwzapnt;ZLnet/minecraft/unmapped/C_vorddnax$C_bfrytpdl;)V
+		ARG 3 hasOverlay
 	METHOD m_hrdduwff getTexture ()Lnet/minecraft/unmapped/C_ncpywfca;
 	METHOD m_jrgzvqyh type ()Lnet/minecraft/unmapped/C_bbljnksk$C_rtwzapnt;
+	METHOD m_kvqjjxxm getOverlayTexture ()Lnet/minecraft/unmapped/C_ncpywfca;
 	CLASS C_rtwzapnt Type
 		FIELD f_melzgwgg textureLocator Ljava/util/function/Function;
 		FIELD f_vzmqsxzm breakSound Lnet/minecraft/unmapped/C_avavozay;

--- a/mappings/net/minecraft/item/ArrowItem.mapping
+++ b/mappings/net/minecraft/item/ArrowItem.mapping
@@ -1,3 +1,2 @@
 CLASS net/minecraft/unmapped/C_qdrvdocr net/minecraft/item/ArrowItem
-	METHOD m_cxaksmmw createProjectile (Lnet/minecraft/unmapped/C_cdctfzbn;Lnet/minecraft/unmapped/C_sddaxwyk;Lnet/minecraft/unmapped/C_usxaxydn;Lnet/minecraft/unmapped/C_sddaxwyk;)Lnet/minecraft/unmapped/C_gstiksvs;
-		ARG 2 stack
+	METHOD m_cxaksmmw getEntity (Lnet/minecraft/unmapped/C_cdctfzbn;Lnet/minecraft/unmapped/C_sddaxwyk;Lnet/minecraft/unmapped/C_usxaxydn;Lnet/minecraft/unmapped/C_sddaxwyk;)Lnet/minecraft/unmapped/C_gstiksvs;

--- a/mappings/net/minecraft/item/AxeItem.mapping
+++ b/mappings/net/minecraft/item/AxeItem.mapping
@@ -2,5 +2,7 @@ CLASS net/minecraft/unmapped/C_qqnexjle net/minecraft/item/AxeItem
 	FIELD f_bwshyjtw STRIPPED_BLOCKS Ljava/util/Map;
 	METHOD m_iwpubrnh handleUseOnBlock (Lnet/minecraft/unmapped/C_cdctfzbn;Lnet/minecraft/unmapped/C_hynzadkk;Lnet/minecraft/unmapped/C_jzrpycqo;Lnet/minecraft/unmapped/C_txtbiemp;)Ljava/util/Optional;
 		ARG 3 player
+	METHOD m_rnmpjycu shouldIgnoreAction (Lnet/minecraft/unmapped/C_yyklzime;)Z
+		ARG 0 context
 	METHOD m_ummyfuic getStrippedState (Lnet/minecraft/unmapped/C_txtbiemp;)Ljava/util/Optional;
 		ARG 1 state

--- a/mappings/net/minecraft/item/CrossbowItem.mapping
+++ b/mappings/net/minecraft/item/CrossbowItem.mapping
@@ -14,6 +14,7 @@ CLASS net/minecraft/unmapped/C_fatxcdqp net/minecraft/item/CrossbowItem
 		ARG 5 speed
 		ARG 6 divergence
 	METHOD m_pfmniexj getSpeed (Lnet/minecraft/unmapped/C_lukqwasx;)F
+		ARG 0 component
 	METHOD m_tgjukcva loadProjectiles (Lnet/minecraft/unmapped/C_usxaxydn;Lnet/minecraft/unmapped/C_sddaxwyk;)Z
 		ARG 0 shooter
 		ARG 1 projectile

--- a/mappings/net/minecraft/item/Item.mapping
+++ b/mappings/net/minecraft/item/Item.mapping
@@ -6,6 +6,8 @@ CLASS net/minecraft/unmapped/C_vorddnax net/minecraft/item/Item
 	FIELD f_jphyibzg LOGGER Lorg/slf4j/Logger;
 	FIELD f_kdmqzdxr translationKey Ljava/lang/String;
 	FIELD f_lfoqoljh recipeRemainder Lnet/minecraft/unmapped/C_vorddnax;
+	FIELD f_nucbkqeo BASE_ATTACK_SPEED Lnet/minecraft/unmapped/C_ncpywfca;
+	FIELD f_puurvgcx BASE_ATTACK_DAMAGE Lnet/minecraft/unmapped/C_ncpywfca;
 	FIELD f_rxraouwr builtInRegistryHolder Lnet/minecraft/unmapped/C_cjzoxshv$C_rjzpeyec;
 	FIELD f_vmipucxo ABSOLUTE_MAX_COUNT I
 	FIELD f_vwakrgmw DEFAULT_MAX_COUNT I
@@ -61,6 +63,8 @@ CLASS net/minecraft/unmapped/C_vorddnax net/minecraft/item/Item
 		ARG 2 world
 		ARG 3 user
 		ARG 4 remainingUseTicks
+	METHOD m_hwznhtnf getEatTicks (Lnet/minecraft/unmapped/C_sddaxwyk;Lnet/minecraft/unmapped/C_usxaxydn;)I
+		ARG 2 entity
 	METHOD m_inqbslnb useOnEntity (Lnet/minecraft/unmapped/C_sddaxwyk;Lnet/minecraft/unmapped/C_jzrpycqo;Lnet/minecraft/unmapped/C_usxaxydn;Lnet/minecraft/unmapped/C_laxmzoqs;)Lnet/minecraft/unmapped/C_ozuepbyj;
 		ARG 1 stack
 		ARG 2 user
@@ -219,6 +223,7 @@ CLASS net/minecraft/unmapped/C_vorddnax net/minecraft/item/Item
 			COMMENT
 			COMMENT @return this instance
 			ARG 1 recipeRemainder
+		METHOD m_pvohmnyq jukeboxPlayable (Lnet/minecraft/unmapped/C_xhhleach;)Lnet/minecraft/unmapped/C_vorddnax$C_bfrytpdl;
 		METHOD m_qvwjjfqj buildAndValidateComponents ()Lnet/minecraft/unmapped/C_kouhnfig;
 		METHOD m_robddprv maxDamage (I)Lnet/minecraft/unmapped/C_vorddnax$C_bfrytpdl;
 			COMMENT Sets the maximum durability of any item configured with this Settings instance.

--- a/mappings/net/minecraft/item/Item.mapping
+++ b/mappings/net/minecraft/item/Item.mapping
@@ -229,7 +229,8 @@ CLASS net/minecraft/unmapped/C_vorddnax net/minecraft/item/Item
 			COMMENT
 			COMMENT @return this instance
 			ARG 1 recipeRemainder
-		METHOD m_pvohmnyq jukeboxPlayable (Lnet/minecraft/unmapped/C_xhhleach;)Lnet/minecraft/unmapped/C_vorddnax$C_bfrytpdl;
+		METHOD m_pvohmnyq jukeboxSong (Lnet/minecraft/unmapped/C_xhhleach;)Lnet/minecraft/unmapped/C_vorddnax$C_bfrytpdl;
+			ARG 1 song
 		METHOD m_qvwjjfqj buildAndValidateComponents ()Lnet/minecraft/unmapped/C_kouhnfig;
 		METHOD m_robddprv maxDamage (I)Lnet/minecraft/unmapped/C_vorddnax$C_bfrytpdl;
 			COMMENT Sets the maximum durability of any item configured with this Settings instance.

--- a/mappings/net/minecraft/item/ItemGroups.mapping
+++ b/mappings/net/minecraft/item/ItemGroups.mapping
@@ -10,8 +10,10 @@ CLASS net/minecraft/unmapped/C_debvfuic net/minecraft/item/ItemGroups
 	FIELD f_pzahtrqp OPERATOR_UTILITIES Lnet/minecraft/unmapped/C_xhhleach;
 	FIELD f_qapwqnww COMBAT Lnet/minecraft/unmapped/C_xhhleach;
 	FIELD f_qcxervyf FOOD_AND_DRINKS Lnet/minecraft/unmapped/C_xhhleach;
+	FIELD f_ryhjstby ITEM_SEARCH Lnet/minecraft/unmapped/C_ncpywfca;
 	FIELD f_sdiyzpyx NATURAL_BLOCKS Lnet/minecraft/unmapped/C_xhhleach;
 	FIELD f_skbtirit PAINTING_SIZE_COMPARATOR Ljava/util/Comparator;
+	FIELD f_xxtxsdvx INVENTORY Lnet/minecraft/unmapped/C_ncpywfca;
 	FIELD f_yqelfgst REDSTONE_BLOCKS Lnet/minecraft/unmapped/C_xhhleach;
 	FIELD f_yqiiybfa SURVIVAL_INVENTORY Lnet/minecraft/unmapped/C_xhhleach;
 	FIELD f_zjfnhgwd FUNCTIONAL_BLOCKS Lnet/minecraft/unmapped/C_xhhleach;
@@ -23,6 +25,8 @@ CLASS net/minecraft/unmapped/C_debvfuic net/minecraft/item/ItemGroups
 	METHOD m_ehiehxzh getFireworkRocketEntries (Lnet/minecraft/unmapped/C_lnokcayd$C_wnogvxsu;Lnet/minecraft/unmapped/C_lnokcayd$C_tndlmady;)V
 		ARG 0 collector
 		ARG 1 visibility
+	METHOD m_fimfsjdv generateEnchantmentEntries (Lnet/minecraft/unmapped/C_lnokcayd$C_wnogvxsu;Lnet/minecraft/unmapped/C_vtbxyypo;Lnet/minecraft/unmapped/C_lnokcayd$C_tndlmady;)V
+		ARG 1 enchantmentLookup
 	METHOD m_ftyxvkzn (Lnet/minecraft/unmapped/C_lnokcayd$C_yymolfee;Lnet/minecraft/unmapped/C_lnokcayd$C_wnogvxsu;)V
 		ARG 0 params
 	METHOD m_gdihuldi tryReloadEntries (Lnet/minecraft/unmapped/C_czxxrbcp;ZLnet/minecraft/unmapped/C_vtbxyypo$C_etmlgbig;)Z
@@ -30,6 +34,8 @@ CLASS net/minecraft/unmapped/C_debvfuic net/minecraft/item/ItemGroups
 		ARG 1 hasPermissions
 		ARG 2 lookupProvider
 	METHOD m_gsotsemd getDisplayableGroups ()Ljava/util/List;
+	METHOD m_jjegmdcl generateMaxLevelEnchantmentEntries (Lnet/minecraft/unmapped/C_lnokcayd$C_wnogvxsu;Lnet/minecraft/unmapped/C_vtbxyypo;Lnet/minecraft/unmapped/C_lnokcayd$C_tndlmady;)V
+		ARG 1 enchantmentLookup
 	METHOD m_kbmenyai generateInstrumentEntries (Lnet/minecraft/unmapped/C_lnokcayd$C_wnogvxsu;Lnet/minecraft/unmapped/C_vtbxyypo;Lnet/minecraft/unmapped/C_vorddnax;Lnet/minecraft/unmapped/C_ednuhnnn;Lnet/minecraft/unmapped/C_lnokcayd$C_tndlmady;)V
 		ARG 0 collector
 		ARG 1 instrumentLookup
@@ -63,6 +69,8 @@ CLASS net/minecraft/unmapped/C_debvfuic net/minecraft/item/ItemGroups
 		ARG 4 featureFlags
 	METHOD m_wgmcrzkx (Lnet/minecraft/unmapped/C_cjzoxshv;)Z
 		ARG 0 variant
+	METHOD m_wnbuxxbk generatePaintingVariantEntries (Lnet/minecraft/unmapped/C_lnokcayd$C_wnogvxsu;Lnet/minecraft/unmapped/C_vtbxyypo$C_etmlgbig;Lnet/minecraft/unmapped/C_vtbxyypo$C_drwjtlvl;Ljava/util/function/Predicate;Lnet/minecraft/unmapped/C_lnokcayd$C_tndlmady;)V
+		ARG 2 paintingVariantLookup
 	METHOD m_xiomsyzz (Lnet/minecraft/unmapped/C_lnokcayd$C_yymolfee;Lnet/minecraft/unmapped/C_lnokcayd$C_wnogvxsu;)V
 		ARG 0 params
 	METHOD m_xtbalece createRegistryKey (Ljava/lang/String;)Lnet/minecraft/unmapped/C_xhhleach;

--- a/mappings/net/minecraft/item/ItemStack.mapping
+++ b/mappings/net/minecraft/item/ItemStack.mapping
@@ -89,6 +89,7 @@ CLASS net/minecraft/unmapped/C_sddaxwyk net/minecraft/item/ItemStack
 	METHOD m_dyvmnknh fromNbt (Lnet/minecraft/unmapped/C_vtbxyypo$C_etmlgbig;Lnet/minecraft/unmapped/C_hhlwcnih;)Lnet/minecraft/unmapped/C_sddaxwyk;
 		ARG 0 lookupProvider
 	METHOD m_ehlafaks copyAndClear ()Lnet/minecraft/unmapped/C_sddaxwyk;
+	METHOD m_ejzzxfrl getItemComponents ()Lnet/minecraft/unmapped/C_kouhnfig;
 	METHOD m_emygxjdg postMine (Lnet/minecraft/unmapped/C_cdctfzbn;Lnet/minecraft/unmapped/C_txtbiemp;Lnet/minecraft/unmapped/C_hynzadkk;Lnet/minecraft/unmapped/C_jzrpycqo;)V
 		ARG 1 world
 		ARG 2 state
@@ -183,7 +184,10 @@ CLASS net/minecraft/unmapped/C_sddaxwyk net/minecraft/item/ItemStack
 			COMMENT the amount to increment
 	METHOD m_nosilizp isDamaged ()Z
 	METHOD m_nyilezsd encode (Lnet/minecraft/unmapped/C_vtbxyypo$C_etmlgbig;Lnet/minecraft/unmapped/C_oivssbvb;)Lnet/minecraft/unmapped/C_oivssbvb;
+	METHOD m_olrjfwfo validate (Lnet/minecraft/unmapped/C_sddaxwyk;)Lcom/mojang/serialization/DataResult;
 	METHOD m_oszmcroz onItemEntityDestroyed (Lnet/minecraft/unmapped/C_uqpzijng;)V
+		ARG 1 entity
+	METHOD m_othdzzkw (Lnet/minecraft/unmapped/C_usxaxydn;Lnet/minecraft/unmapped/C_jzrpycqo;)V
 		ARG 1 entity
 	METHOD m_phvcusub appendModifierTooltips (Ljava/util/function/Consumer;Lnet/minecraft/unmapped/C_jzrpycqo;)V
 		ARG 1 textConsumer
@@ -274,6 +278,8 @@ CLASS net/minecraft/unmapped/C_sddaxwyk net/minecraft/item/ItemStack
 	METHOD m_xqyyeqdc setHolder (Lnet/minecraft/unmapped/C_astfners;)V
 		ARG 1 holder
 	METHOD m_xvzvtufo getName ()Lnet/minecraft/unmapped/C_rdaqiwdt;
+	METHOD m_yaszceoi validateComponents (Lnet/minecraft/unmapped/C_kouhnfig;)Lcom/mojang/serialization/DataResult;
+		ARG 0 components
 	METHOD m_yawcimjo getEntityHolder ()Lnet/minecraft/unmapped/C_astfners;
 	METHOD m_yprgpdac takesDamageFrom (Lnet/minecraft/unmapped/C_sbxfkpyv;)Z
 		ARG 1 damageSource

--- a/mappings/net/minecraft/item/ItemStack.mapping
+++ b/mappings/net/minecraft/item/ItemStack.mapping
@@ -81,7 +81,7 @@ CLASS net/minecraft/unmapped/C_sddaxwyk net/minecraft/item/ItemStack
 	METHOD m_cgmsskmm split (I)Lnet/minecraft/unmapped/C_sddaxwyk;
 		ARG 1 amount
 	METHOD m_cgmunnul copyComponentsToNewStack (Lnet/minecraft/unmapped/C_gmbqjnle;)Lnet/minecraft/unmapped/C_sddaxwyk;
-		COMMENT {@return a new item stack with the components and stack size copied from this stack, unless this stack is empty, then returns an empty item stack}
+		COMMENT {@return a new item stack with the components and stack size copied from this stack. If this stack is empty, returns an empty item stack}
 		COMMENT
 		COMMENT @see #copyComponentsToNewStack
 		COMMENT the resulting item stack's item

--- a/mappings/net/minecraft/item/ItemUsage.mapping
+++ b/mappings/net/minecraft/item/ItemUsage.mapping
@@ -8,6 +8,7 @@ CLASS net/minecraft/unmapped/C_hcavgpph net/minecraft/item/ItemUsage
 		ARG 3 creativeOverride
 	METHOD m_jtqrwfrz spawnItemContents (Lnet/minecraft/unmapped/C_uqpzijng;Ljava/lang/Iterable;)V
 		ARG 0 itemEntity
+		ARG 1 itemstacks
 	METHOD m_ryjfhcxq exchangeStack (Lnet/minecraft/unmapped/C_sddaxwyk;Lnet/minecraft/unmapped/C_jzrpycqo;Lnet/minecraft/unmapped/C_sddaxwyk;)Lnet/minecraft/unmapped/C_sddaxwyk;
 		ARG 0 inputStack
 		ARG 1 player

--- a/mappings/net/minecraft/item/MaceItem.mapping
+++ b/mappings/net/minecraft/item/MaceItem.mapping
@@ -20,3 +20,5 @@ CLASS net/minecraft/unmapped/C_dromlfke net/minecraft/item/MaceItem
 	METHOD m_tazjgygw knockbackNearbyEntities (Lnet/minecraft/unmapped/C_cdctfzbn;Lnet/minecraft/unmapped/C_jzrpycqo;Lnet/minecraft/unmapped/C_astfners;)V
 		ARG 1 attacker
 		ARG 2 target
+	METHOD m_yidgbebs (Lnet/minecraft/unmapped/C_usxaxydn;)Z
+		ARG 0 entity

